internal/expr/bindtypes.go

```diff
@@ -143,17 +145,28 @@ func bindInputTypes(ti typeNameToInfo, e *inputExpr) (tm typeinfo.Member, err er
 		return nil, typeMissingError(e.sourceType.typeName, getKeys(ti))
 	}
 
-	tm, err = info.TypeMember(e.sourceType.memberName)
+	tm, err := info.TypeMember(e.sourceType.memberName)
 	if err != nil {
 		return nil, err
 	}
-	return tm, nil
+	return &typedInputExpr{tm}, nil
 }
 
-// bindOutputTypes binds the output expression to concrete types. It then checks
-// the expression is formatted correctly and generates the columns for the query
-// and the typeMembers the columns correspond to.
-func bindOutputTypes(ti typeNameToInfo, e *outputExpr) (outCols []columnAccessor, typeMembers []typeinfo.Member, err error) {
+// outputExpr represents a named target output variable in the SQL expression,
+// as well as the source table and column where it will be read from.
```

Comment:
```gp
// outputExpr represents a named target output variable in the SQL expression.
// It contains the source table and column where it will be read from and the value
// where it will be read to.
```
It does not **represent** the source table and column ...